src/test/java/api/loans/CheckOutByBarcodeTests.java

```diff
@@ -722,4 +723,158 @@ public void canCheckOutUsingProxyUserBarcodeThatContainsSpaces()
 
     assertThat(smallAngryPlanet, hasItemStatus(CHECKED_OUT));
   }
+
+  @Test
+  public void canCheckOutOnOrderItem()
+    throws MalformedURLException,
+    InterruptedException,
+    TimeoutException,
+    ExecutionException {
+
+    IndividualResource smallAngryPlanet = itemsFixture.basedUponSmallAngryPlanet(item -> item.onOrder());
+
+    final IndividualResource jessica = usersFixture.jessica();
+
+    final IndividualResource response = loansFixture.checkOutByBarcode(
+      new CheckOutByBarcodeRequestBuilder()
+        .forItem(smallAngryPlanet)
+        .to(jessica)
+        .at(servicePointsFixture.cd1()));
+
+    final JsonObject loan = response.getJson();
+
+    assertThat(loan.getString("id"), is(notNullValue()));
+
+    assertThat("user ID should match barcode",
+      loan.getString("userId"), is(jessica.getId()));
+
+    assertThat("item ID should match barcode",
+      loan.getString("itemId"), is(smallAngryPlanet.getId()));
+
+    assertThat("status should be open",
+      loan.getJsonObject("status").getString("name"), is("Open"));
+
+    smallAngryPlanet = itemsClient.get(smallAngryPlanet);
+
+    assertThat(smallAngryPlanet, hasItemStatus(CHECKED_OUT));
+  }
+
+  @Test
+  public void canCheckOutOnOrderItemWithRequest()
+    throws MalformedURLException,
+    InterruptedException,
+    TimeoutException,
+    ExecutionException {
+
+    IndividualResource smallAngryPlanet = itemsFixture.basedUponSmallAngryPlanet(item -> item.onOrder());
```

Comment:
Alternatively, this can use a method reference
```suggestion
    IndividualResource smallAngryPlanet = itemsFixture.basedUponSmallAngryPlanet(ItemBuilder::onOrder);
```

Which do you prefer?